sourmash/lca/command_index.py

```diff
@@ -78,18 +84,21 @@ def load_taxonomy_assignments(filename, delimiter=',', start_column=2,
 
 
 def generate_report(record_duplicates, record_no_lineage, record_remnants,
-                    unused_lineages, filename):
+                    unused_lineages, unused_identifiers, filename):
```

Comment:
This bothers me a bit, but since it's an internal function I will let it pass.

But, if it was a function available on the API: since all arguments are positional, it would break calling this in other functions when you change the number of arguments you're passing (and, even more, the 5th argument changed meaning, it's not a filename anymore!). 

(An example of the things we could do if we stopped supporting Python 2:
https://www.asmeurer.com/python3-presentation/slides.html#8 )

Reply:
agree with comment.